### ISSUE TITLE
Image scaling doesn't work for RGB565A8 format

### ISSUE THREAD

**Issue by sirius506** [kisvegabor/lvgl#5537]
### LVGL version

v9.0.0

### What happened?

Music player demo uses ARGB8888 format album cover image. I have generated RGB585A8 format cover image file and rebuilt the demo. Now, cover image crashed when it is zoomed.

### How to reproduce?

```
cd demos/music/assets
LVGLImage.py --ofmt C --cf RGB565A8 png/272_png/cover_1.png
mv output/cover_1.c ../img_lv_demo_music_cover1.c
```

**Comment by kisvegabor** [comment]
I can reproduce it and will look into it next week.

**Comment by kisvegabor** [comment]
Sorry, I'll get back to this ASAP!

**Comment by kisvegabor** [comment]
Now it works on my end with the latest LVGL from master.

```c
LV_IMAGE_DECLARE(cover_111);
lv_obj_t * img1 = lv_image_create(lv_screen_active());
lv_image_set_src(img1, &cover_111);
lv_obj_align(img1, LV_ALIGN_CENTER, 0, 0);
lv_image_set_scale(img1, 400);
```
 
![image](https://github.com/lvgl/lvgl/assets/7599318/c501441c-1d81-4c4d-b7dc-366f1240f8ef)

[cover111.c.zip](https://github.com/lvgl/lvgl/files/14393986/cover111.c.zip)

**Comment by sirius506** [comment]
It looks image scaling itself is working, but music demo does't.

I am testing music demo with lv_port_stm32f769_disco. With latest master and RGB565A8 image, I got this result.  (Sorry, my screen is dirty, please ignore black dashes on below screen part.)

![IMG_20240225_120953](https://github.com/lvgl/lvgl/assets/12758516/7545b3e0-0810-44c6-b83e-240c7d58d474)

Cover image still goes wrong when scaled and play button is no longer displayed correctly.  With v9.0.0 release code, play button has shown correctly.

**Comment by sirius506** [comment]
OK, I confirmed #5782 fixed the image crash problem.
Now, demo screen looks like this.

![IMG_20240304_100703](https://github.com/lvgl/lvgl/assets/12758516/5d1f2db7-b213-4d1e-b3d9-831cd16df746)

The cover image is shown as square figure. I checked PNG image in assets and its background alpha value is 0xff. So, the fix sets correct alpha value.

To get same cover image effect as we saw in older releases, I think PNG cover images should be fixed to have transparent background. But, I wonder why and how it has been working correctly before...
